modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -32,19 +50,34 @@ export default class NestedAccordion extends Base {
 		};
 	}
 
+	getKeyboardNavigationSettings() {
+		return this.getSettings();
+	}
+
 	onInit( ...args ) {
 		super.onInit( ...args );
 
 		if ( elementorFrontend.isEditMode() ) {
 			this.interlaceContainers();
 		}
+
+		if ( 'nested-accordion.default' === this.getSettings( 'elementName' ) ) {
+			const handler = new elementorModules.frontend.handlers.NestedTitleKeyboardHandler( this.getKeyboardNavigationSettings() );
+			handler.handeTitleLinkEnterOrSpaceEvent = this.clickListener.bind( this ); // To support enter and space keys
+		}
 	}
 
 	interlaceContainers() {
 		const { $contentContainers, $accordionItems } = this.getDefaultElements();
 
 		$contentContainers.each( ( index, element ) => {
 			$accordionItems[ index ].appendChild( element );
+
+			const summary = $accordionItems[ index ].querySelector( 'summary' ),
+				itemId = summary.getAttribute( 'aria-controls' );
+
+			element.setAttribute( 'role', 'region' );
```

Comment:
Is this attribute always the same?
If so, why don't we set this inside the render() function?